docs/scheduled-transactions/revised-spec.md

```diff
@@ -0,0 +1,157 @@
+  
+```  
+message ScheduleCreateTransactionBody {  
+  SchedulableTransactionBody scheduledTransactionBody = 1; // The scheduled transaction; its TransactionID must not be set
+  string memo = 2; // An optional memo with a UTF-8 encoding of no more than 100 bytes
+  Key adminKey = 3; // An optional Hedera key which can be used to sign a `ScheduleDelete` and remove the schedule
+  bool executeImmediatelyOnValidSignatures = 4; // May the scheduled transaction be executed as soon as its schedule has enough signing keys? MUST be `true`.
+  AccountID scheduledPayerAccountID = 5; // The id of the account to be charged the service fee for the scheduled transaction at the consensus time that it executes (if ever).
+}  
+```  
+
+The new `SchedulableTransactionBody` message is a strict subset of the `TransactionBody` message which omits the
+top-level `TransactionID`, `nodeAccountID`, and `transactionValidDuration` fields; and does not allow the `ScheduleCreateTransactionBody`
+and `ScheduleSignTransactionBody` messages in its `data` element.
+
+In release `0.13.0`, the fourth field has incomplete support, as stated in the comments above. That is, 
+ - Only schedules that permit immediate execution of their scheduled transaction given enough signing keys may be created. 
+
+As with all other entity types, a schedule remains in network state until it expires, even if its scheduled 
+transaction has already been executed.
+
+### Paying for scheduled transactions
+
+If the `ScheduleCreate` gives a `scheduledPayerAccountID`, the network will charge this payer the service fee 
+for the scheduled transaction at the consensus time that it executes (if ever). If no such payer is specified, 
+the network will charge the payer of the originating `ScheduleCreate`.
+
+### Receipts and duplicate creations
+
+When a `ScheduleCreate` resolves to `SUCCESS`, its receipt includes a new `ScheduleID` field
+with the id of the created schedule. Its receipt _also_ includes a new 
+`scheduledTransactionID` field with the `TransactionID` that can be used to query for the 
+record of the scheduled transaction's execution. 
+
+This `scheduledTransactionID` will be the `TransactionID` of the `ScheduleCreate`, 
+with a new additional field `scheduled=true`.  However, clients should always use the id 
+from the receipt instead of relying on this correspondence.
+
+#### Duplicate creations
+There is a special case in which a `ScheduleCreate` transaction tries to re-create a
+schedule that already exists in state. When this happens, the `ScheduleCreate` resolves 
+to a new status of `IDENTICAL_SCHEDULE_ALREADY_CREATED`, and the `ScheduleID` in its receipt points
+to the existing schedule. A client receiving `IDENTICAL_SCHEDULE_ALREADY_CREATED` can
+then submit a `ScheduleSign` (see below) with the given `ScheduleID`, signing with
+the same Ed25519 keys it used for its own create attempt. 
+
+Note the `scheduledPayerAccountID` field does not affect the identity of a schedule;
```

Comment:
I think this definition of equality should be explicitly mentioned here. It should exactly list which fields are included and omitted. It should also mention it's not relying on the non-deterministic protobuf serialization but using Java's equal/hashCode.

I'm also interested to know how unknown fields will be handled with this equality comparison.